pkg/kubectl/cmd/config/set.go

```diff
@@ -62,12 +63,8 @@ func NewCmdConfigSet(out io.Writer, configAccess clientcmd.ConfigAccess) *cobra.
 				return
 			}
 
-			err := options.run()
-			if err != nil {
-				fmt.Fprintf(out, "%v\n", err)
-			} else {
-				fmt.Fprintf(out, "property %q set.\n", options.propertyName)
-			}
+			cmdutil.CheckErr(options.run())
+			fmt.Fprintf(out, "property %q set.\n", options.propertyName)
```

Comment:
Generally we would only use a period at the end if we were making this a sentence, which would mean we would capitalize the first letter.  I would recommend making all of these fit that model:

    Property %q set.\n